fission-cli/library/Fission/CLI/Handler/User/Whoami.hs

```diff
@@ -67,5 +67,5 @@ whoami = do
               DecodeFailure _ _ -> "Trouble decoding the registration response."
               _                 -> "Invalid content type."
 
-      UTF8.putText "Please contact Fission support at https://fission.codes or delete `~/.ssh/fission` and try again."
+      UTF8.putText "Please contact Fission support at https://fission.codes or delete `~/.config/fission` and try again."
```

Comment:
This is great 👏 And to echo James — great catch!

You know, we could go one step further and hook it into our standard paths and display that:

https://github.com/fission-suite/fission/blob/5d0a808adc3e3cffb1c4eae17ab1f5194bc9814e/fission-cli/library/Fission/CLI/Environment/Class.hs#L7

We keep the standard paths inside the config in [`Fission.CLI.Environment.Path`](https://github.com/fission-suite/fission/blob/5d0a808adc3e3cffb1c4eae17ab1f5194bc9814e/fission-cli/library/Fission/CLI/Environment/Path.hs) module

Reply:
Yeah, makes sense to pull this from a standard path. 

I'll give this a try. Thanks for the hints. :slightly_smiling_face:

Reply:
Alright, a standard path for `globalDir` is added and we are using it in the error message. 

Right now, it will display the absolute path to the directory. Should consider displaying a relative path off of `~` instead?

Reply:
I agree that the `~` version would be prettier, but I'm not really sure how we would detect that and replace it, especially across different OS layouts (e.g. `/Users/expede` vs `/home/expede`). Having it update automatically is more important _for now_, so I think ship it with absolute paths and we can revisit if needed 🚀 

Thanks for the quick turn around on that 👏

Reply:
Sounds good! :+1: